doc/source/index.rst

```diff
@@ -286,6 +286,7 @@ Getting Involved
    rayserve/tutorials/tensorflow-tutorial.rst
    rayserve/tutorials/pytorch-tutorial.rst
    rayserve/tutorials/sklearn-tutorial.rst
+   rayserve/tutorials/batch-tutorial.rst
```

Comment:
I guess I don't totally understand why this needs to be broken out into its own page.

Alternatively, couldn't we break out a section called "Backend Configurations" or something like that and have all of these here? Also, we have batching [here](https://docs.ray.io/en/master/rayserve/overview.html#batching), is this something different? Is it just a more in depth walkthrough of batching?

Reply:
This is a learning-oriented tutorial. The batching in over page is goal-oriented how-to guides.